src/components/searchViewComponents/BaseList.vue

```diff
@@ -114,26 +120,17 @@ const queryPage = computed({
       class="justify-space-between d-flex mb-3"
       style="gap: 5px"
     >
-      <hyperlink
-        :disabled="!anyFileType"
-        :to="{ ...route, query: { ...route.query, type: fileType } }"
-        class="flex-grow-1"
-      >
-        <div
-          class="w-100 d-flex justify-start align-center text-ipfsPrimary-lighten-1 v-btn v-btn--density-default v-btn--size-default v-btn--variant-outlined"
-          color="ipfsPrimary"
+      <hyperlink :disabled="!anyFileType" :to="viewAllTo" class="flex-grow-1">
+        <!-- Note: v-btn has a "to" prop as well, which should eliminate the need for hyperlink here. However, it causes the btn to be rendered as 'tonal', overriding the text variant here -->
```

Comment:
Please suggest another way to refer to undocumented relevant behaviour of this particular component. 

It took quite a bit of experimenting to discover this and this comment is intended to prevent our future selves from having to do these experiments again (trying to optimise out the hyperlink only to discover that `to` for `<v-btn>` changes the rendering.